news/4.36/jdt.html

```diff
@@ -193,7 +193,26 @@ <h2>Debug</h2>
 				</td>
 				</td>
 			</tr>
-
+			<tr id="DefaultFoldingMechanism">
+				<td class="title">New folding mechanism as Default</td>
+				<td class="content">
+					<p>
+						In the previous releases, a preference to enable different kinds of folding was introduced (see the news for <a href="../4.35/jdt.php#new-folding">4.35</a>). 
+						The feature has been further improved since then and is now <b>enabled by default</b>.
+					</p>
+					<p>
+						This feature enhances the code folding mechanism in Eclipse JDT by enabling folding for control statements like if, while, switch and for. </b>
```

Comment:
I'd enclose the structures in `<code>` blocks

```suggestion
						This feature enhances the code folding mechanism in Eclipse JDT by enabling folding for control statements like <code>if</code>, <code>while</code>, <code>switch</code> and <code>for</code>. </b>
```

Reply:
Addressed in #316